### PR TITLE
37 course display

### PR DESCRIPTION
**Pull request by llaulier**

### CODE REVIEW

File: src/components/coursedisplay/CourseDisplayTag.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        .filter((parc) => {
+          return courseId === parc.course_id;
+        })
+        .map((parc) => {
+          return (
+            <Chip
+              className={classes.chip}
+              label={parc.label}
+              variant="outlined"
+              color="primary"
```
Comment (RoxaneDrelon) [inline src/components/coursedisplay/CourseDisplayTag.jsx]: Bad, never used

**Review by RoxaneDrelon — CHANGES_REQUESTED** [review]
Bad unused component

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by RoxaneDrelon — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**